### PR TITLE
creating step function resource test case.

### PR DESCRIPTION
**Pull request by coingraham**
Interested in helping out with this project.  Getting my feet wet adding a step function test case with placebo data.

### CODE REVIEW

**Comment by bclements** [comment]
We always love to get new contributors! @coingraham Can you make sure to sign the individual contributor agreement located at: https://docs.google.com/forms/d/19LpBBjykHPox18vrZvBbZUcK6gQTj7qv1O5hCduAZFU/viewform

Thanks

**Comment by coingraham** [comment]
@bclements, I have already.  How long does it take?

**Comment by kapilt** [comment]
Thanks for the pull request, one thing to note is that we generally replace any account ids recorded with dummy values, outside of some known dedicated testing accounts, as it’s a security concern. Could you  update the recorded outputs here to remove those present here. Wrt to the cla it’s instant, thanks

File: tests/test_sfn.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                        'type': 'value',
+                        'key': 'name',
+                        'value': 'test'
+                    }
+                ]
+            },
+            session_factory=session_factory
+        )
+        resources = p.run()
+        self.assertTrue(len(resources))
```
Comment (kapilt) [inline tests/test_sfn.py]: typically a good unit test will assert the number/cardinality of the resources along with some attribute of the resource (identity typically).

Comment (coingraham) [reply tests/test_sfn.py]: I've updated the first to include 1 return value, added a check that the name should be test.  Also stripped the account number and replaced with a fake one I've seen used elsewhere.

**Comment by kapilt** [comment]
there's one more account id in list machines recorded output. we should probably add a developer utility to automate this for folks.

**Comment by coingraham** [comment]
Doh, my bad.  I'll fix and push in a minute.  Good looking out.

**Pull request merged**